### PR TITLE
[FLINK-28300] Fix helm lint invalid Yaml document separator

### PR DESCRIPTION
**Pull request by MaksimAniskov**
Fixes FLINK-28300

### CODE REVIEW

**Comment by gyfora** [comment]
Thank you @MaksimAniskov , next time when you open a PR please tag both the PR and the commit with the jira like : `[FLINK-XXXX] My Commit/PR message` this way it's easier for reviewers.

you can find details about the process for more complex changes here: https://flink.apache.org/contributing/contribute-code.html

File: helm/flink-kubernetes-operator/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 ################################################################################
 
 ---
-{{- if .Values.operatorServiceAccount.create -}}
+{{if .Values.operatorServiceAccount.create }}
```
[the remark this answers is not in the document]
Comment (morhidi) [reply helm/flink-kubernetes-operator/templates/serviceaccount.yaml:20]: I'd also prefer to keep it `{{- if .Values.operatorServiceAccount.create }}` just to be consistent with the rest of the code

[the remark this answers is not in the document]
Comment (morhidi) [reply helm/flink-kubernetes-operator/templates/serviceaccount.yaml:20]: https://helm.sh/docs/chart_template_guide/control_structures/

**Comment by mbalassi** [comment]
@morhidi will fix the typo and merge tomorrow, thanks for the ping

**Pull request closed without merging**